src/world/components/lib/getRandomTransition.js

```diff
@@ -0,0 +1,7 @@
+const matchTransitionFor = require('./matchTransitionFor');
+
+module.exports = function getRandomTransition(world) {
+  const moveableSet = world.things.filter(thing => thing.locations.length > 1);
+  const randomThing = moveableSet[Math.floor(Math.random() * moveableSet.length)];
+  return randomThing && [matchTransitionFor(randomThing, world.numRules, world.rules), randomThing];
```

Comment:
This is where the issue with single locations came from. If `movableSet` was empty, picking a random member would fail.